src/main/java/com/corndel/cashregister/Level4.java

```diff
@@ -10,6 +10,20 @@ public class Level4 {
    */
   public static boolean canMakeAmount(int target, List<Item> drawer) {
     // TODO
-    return false;
+    int targetAmount = target;
+    int total = Level3.sumDrawer(drawer);
```

Comment:
I like that you are using the logic you have previously written in your other classes